client/blocks/login/continue-as-user.jsx

```diff
@@ -50,41 +56,60 @@ function ContinueAsUser( { currentUser, redirectUrlFromQuery, onChangeAccount }
 	// like that, but it is better than the alternative, and in practice it should happen quicker than
 	// the user can notice.
 
+	const notYouText = isSignUpFlow
+		? translate( 'Not you?{{br/}} Sign out or log in with {{link}}another account{{/link}}', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **27** times:
`translate( 'Not you?{{br/}}Log in with {{link}}another account{{/link}}' )` *ES Score: 6*